### PR TITLE
m.coupang.com - Trackparameters Self-Promo Mobile-app

### PR DESCRIPTION
**Pull request by piquark6046**
### URL
https://m.coupang.com/vm/products/6287450931?itemId=12938486294
https://m.coupang.com/nm/search?q=Ryzen+7+5800X

### How to reproduce
1. Visit [here](https://m.coupang.com/nm/search?q=Ryzen+7+5800X) on Mobile or in responsive design mode.
2. Click one of the displayed products.

### Related PR
https://github.com/AdguardTeam/AdguardFilters/pull/111287

### CODE REVIEW

File: AnnoyancesFilter/sections/mobile-app.txt [hunk 1]
```diff
@@ -22,6 +22,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-openInAppButton
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
+m.coupang.com#$#body > #fullBanner { display: none !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/mobile-app.txt:25]: Why not `m.coupang.com###fullBanner`?

File: AnnoyancesFilter/sections/self-promo.txt [hunk 2]
```diff
@@ -58,6 +58,8 @@ disk.yandex.*##.mail360-banner
 @@||yastatic.net/daas/atom.js$domain=mail.yandex.by|mail.yandex.kz|mail.yandex.ru|mail.yandex.ua|mail.yandex.com|mail.yandex.com.tr
 !-----------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/pull/111386
+m.coupang.com###buy-button > #sdp-bottom-promotion-banner-sign-up
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:62]:
```
m.coupang.com###buy-button > div#sdp-bottom-promotion-banner-sign-up
```
Also it's a popup, so you should move it to this file.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/mobile-app.txt (hunk 1 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/mobile-app.txt]: Pushed.

File: AnnoyancesFilter/sections/self-promo.txt (hunk 2 above)
Comment (piquark6046) [reply AnnoyancesFilter/sections/self-promo.txt]: Pushed.

**Review by zloyden — APPROVED** [review]

**Pull request merged**